src/shortcuts/delete.ts

```diff
@@ -31,6 +32,7 @@ const exec: Shortcut['exec'] = (dispatch, getState, e) => {
   } else if (findDescendant(state, head(cursor), '=readonly')) {
     dispatch(error({ value: `"${ellipsize(value)}" is read-only and cannot be deleted.` }))
   } else {
+    editingValueStore.update(null)
```

Comment:
When a thought is deleted, the cursor moves to the next thought. Unless it is the last thought in the thoughtspace, there is never a point where the editingValue becomes `null`, so this is not correct.

It is triggering a re-render, which fixes the observed issue, but through incorrect means. Setting an invalid editingValue could leads to regressions in other parts of the app that expect the editingValue to reflect the value of the cursor thought that is being edited.